File: 백준/Silver/2941. 크로아티아 알파벳/크로아티아 알파벳.java
import java.io.*;

public class Main {

    private static String[] alphas = new String[]{"c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z="};

    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String input = br.readLine();

        for(String alpha : alphas) {
            if(input.contains(alpha)) input = input.replace(alpha, "!");
        }
        
        System.out.println(input.length());
    }
}

File: 백준/Silver/2941. 크로아티아 알파벳/크로아티아 알파벳.kt
fun main() = with(System.`in`.bufferedReader()) {

    val alphas = arrayOf("c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z=");
    var input = readLine()!!;

    alphas.forEach { alpha ->
        if( input.contains(alpha)) input = input.replace(alpha, "!");
    }

    println(input.length);
}